### PR TITLE
Some misc cleanup

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

**Review by BenBE — COMMENTED** [review]
In the vector implementation there are 3 different implementations for sorting algorithms present and it would be nice if we could stick with only one; or better yet replace them with something that avoids the worst case on sorted data (Quicksort is particularly bad here). Would be good to see if Merge Sort or better even [Timsort](https://en.wikipedia.org/wiki/Timsort) could solely be used (Maybe extra issue for this rework?).

File: CPUMeter.c [hunk 1]
```diff
@@ -16,7 +16,7 @@ in the source distribution for its full text.
 #include <string.h>
 #include <math.h>
 
-int CPUMeter_attributes[] = {
+static const int CPUMeter_attributes[] = {
    CPU_NICE, CPU_NORMAL, CPU_SYSTEM, CPU_IRQ, CPU_SOFTIRQ, CPU_STEAL, CPU_GUEST, CPU_IOWAIT
```
[the remark this answers is not in the document]
Comment (cgzones) [reply CPUMeter.c]: done

File: MemoryMeter.c [hunk 2]
```diff
@@ -17,7 +17,7 @@ in the source distribution for its full text.
 #include <assert.h>
 
 
-int MemoryMeter_attributes[] = {
+static const int MemoryMeter_attributes[] = {
    MEMORY_USED, MEMORY_BUFFERS, MEMORY_CACHE
```
[the remark this answers is not in the document]
Comment (cgzones) [reply MemoryMeter.c]: done

File: TasksMeter.c [hunk 3]
```diff
@@ -11,7 +11,7 @@ in the source distribution for its full text.
 #include "CRT.h"
 
 
-int TasksMeter_attributes[] = {
+static const int TasksMeter_attributes[] = {
    CPU_SYSTEM, PROCESS_THREAD, PROCESS, TASKS_RUNNING
```
[the remark this answers is not in the document]
Comment (cgzones) [reply TasksMeter.c]: done

File: LoadAverageMeter.c [hunk 4]
```diff
@@ -11,11 +11,13 @@ in the source distribution for its full text.
 #include "Platform.h"
 
 
-int LoadAverageMeter_attributes[] = {
+static const int LoadAverageMeter_attributes[] = {
    LOAD_AVERAGE_ONE, LOAD_AVERAGE_FIVE, LOAD_AVERAGE_FIFTEEN
```
[the remark this answers is not in the document]
Comment (cgzones) [reply LoadAverageMeter.c]: done

**Pull request merged**